cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -574,10 +581,27 @@ def compose_trajectory(self, component_name, parameter_name):
 			traj_msg.points.append(point_msg)
 		return (traj_msg, 0)
 
-	def calculate_point_time(self, component_name, start_pos, end_pos, default_vel):
+	def calculate_point_time(elf, component_name, start_pos, end_pos, default_vel, default_acc):
 		try:
 			d_max = max(list(abs(numpy.array(start_pos) - numpy.array(end_pos))))
-			point_time = max(d_max / default_vel, 0.4)	# use minimal point_time
+			t1 = default_vel / default_acc
+			s1 = default_acc / 2 * t1**2
+			if (2 * s1 < d_max):
+				# with constant velocity phase (acc, const vel, dec)
+				# 1st phase: accelerate from v=0 to v=default_vel with a=default_acc in t=t1
+				# 2nd phase: constante velocity with v=default_vel and t=t2
+				# 3rd phase: decceleration (analog to 1st phase)
+				s2 = d_max - 2 * s1
+				t2 = s2 / default_vel
+				t = 2 * t1 + t2
+			else:
+				# without constant velocity phase (only acc and dec)
+				# 1st phase: accelerate from v=0 to v=default_vel with a=default_acc in t=t1
+				# 2nd phase: missing because distance is to short (we already reached the distance with the acc and dec phase)
+				# 3rd phase: decceleration (analog to 1st phase)
+				t = math.sqrt(d_max / default_acc)
+			point_time = max(t, 0.4)	# use minimal point_time
+			print point_time
```

Comment:
remove the `print` output

Reply:
done in https://github.com/ipa320/cob_command_tools/pull/227/commits/c44d4450ea11ae3ff399ec458251f25fb1553d95